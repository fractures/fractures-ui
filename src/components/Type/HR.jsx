import PropTypes from "prop-types"
import styled from "styled-components"

const propTypes = {
	className: PropTypes.string
}

const HR = styled.hr`
	margin-bottom: 0;
	margin-top: 0;

	border: 0;
	border-bottom: 1px solid var(--color-gray);
`

HR.propTypes = propTypes

export default HR
