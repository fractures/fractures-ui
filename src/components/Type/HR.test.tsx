import HR from './HR'

describe('HR', () => {
	const tree = renderer.create(<HR />).toJSON()

	it('Matches snapshot', () => {
		expect(tree).toMatchSnapshot()
	})
})
