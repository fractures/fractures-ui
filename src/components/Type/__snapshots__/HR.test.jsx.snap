// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HR Matches snapshot 1`] = `
<hr
  className="sc-bdVaJa cLgotO"
/>
`;
